h2/src/docsrc/html/features.html

```diff
@@ -959,6 +959,7 @@ <h3>MariaDB Compatibility Mode</h3>
 </li><li>LIMIT / OFFSET clauses are supported.
 </li><li>AUTO_INCREMENT clause can be used.
 </li><li>YEAR data type is treated like SMALLINT data type.
+</li><li>Use index in GROUP BY clause to refer to the column position in the SELECT list.
```

Comment:
GROUP BY clause can contain 1-based positions of expressions from the SELECT list.